test/test-execute/exec-privatedevices-no-capibility-mknod.service

```diff
@@ -0,0 +1,7 @@
+[Unit]
+Description=Test CAP_MKNOD capability for PrivateDev=yes
+
+[Service]
+PrivateDevices=no
+ExecStart=/bin/sh -x -c 'c=$$(capsh --print | grep "cap_mknod"); test -n "$$c"'
```

Comment:
I don't know it it's a useful cleanup, but you could just do: `capsh --print | grep cap_mknod`, and `! capsh --print | grep cap_mknod` below.